judging/generate_predictions_local.py

```diff
@@ -83,11 +102,17 @@ def do_main():
                         required=True,
                         help="Path to the python script that should be run to generate predictions. According to the "
                              "API conversion this script should be named predict.py")
+    parser.add_argument("-v", "--validation-module",
```

Comment:
New parameter so driver script can pass in location of the validation module. Bit clunky but it works. More elegant solution -- make sure the module is in `PYTHONPATH` and invoke via `python -m`. But that's more work.